node/src/components/consensus/protocols/highway/synchronizer/tests.rs

```diff
@@ -216,7 +213,7 @@ fn do_not_download_synchronized_dependencies() {
             "unexpected dependency request {:?}",
             outcomes
         );
-        let pv_dep = maybe_pv.unwrap().vertex().id();
+        let pv_dep = pv.vertex().id();
         assert!(units.contains(&pv_dep), "unexpected dependency");
```

Comment:
Why not remove it, so we also check that there are no duplicates?
```suggestion
        assert!(units.remove(&pv_dep), "unexpected dependency");
```

Reply:
Done in [`12bb6c2` (#1312)](https://github.com/CasperLabs/casper-node/pull/1312/commits/12bb6c28701da5269fcc8d5cec73b2509afa6f36)